src/components/layout/header/HeaderProfile/index.tsx

```diff
@@ -14,20 +14,28 @@ import styles from './HeaderProfile.module.scss';
 
 const cx = classNames.bind(styles);
 
+const { top, bottom } = SVGS.arrow;
+
 type HeaderProfileProps = {
   nickname: string;
   profileImageUrl: string;
   deviceType: DeviceType;
   isActivated: boolean;
   onClick: MouseEventHandler<HTMLButtonElement>;
+  headerProfileRef: React.RefObject<HTMLButtonElement>;
```

Comment:
상단에 이미 react 를 임포트 하고있으니까, React는 임포트문에 추가하고 여기서는 아래처럼 써줘도 좋을거같습니다!
```suggestion
  headerProfileRef: RefObject<HTMLButtonElement>;
```

Reply:
수정했습니다!